src/granite.cr

```diff
@@ -2,6 +2,7 @@ require "yaml"
 require "db"
 
 module Granite
+  TIME_ZONE = "UTC"
```

Comment:
I recommend either using vscode crystal plugin or vim plugin to automatically format the code.  or run `crystal tool format .` to fix the tests.

Reply:
Updated